lib/internal/test_runner/coverage.js

```diff
@@ -338,9 +339,10 @@ class TestCoverage {
 
 
   mapCoverageWithSourceMap(coverage) {
+    const sourceMapEnabled = getOptionValue('--enable-source-maps');
     const { result } = coverage;
     const sourceMapCache = coverage['source-map-cache'];
-    if (!sourceMapCache) {
+    if (!sourceMapEnabled || !sourceMapCache) {
```

Comment:
Don't use getOptionValue. Instead, check whether the test runners sourceMaps is true.

Reply:
@RedYetiDev , do you have any clue on how I could do that?

Reply:
See my previous implementation for more information